examples/games_fps.html

```diff
@@ -422,6 +422,16 @@
 
 			} );
 
+			function teleportPlayerIfOob(){
+				if (camera.position.y <= -25){
+					playerCollider.start = new THREE.Vector3( 0, 0.35, 0 );
+					playerCollider.end = new THREE.Vector3( 0, 1, 0 );
```

Comment:
Instead of creating new objects, do this:
```js
playerCollider.start.set( 0, 0.35, 0 );
playerCollider.end.set( 0, 1, 0 );
```

Reply:
Thanks! I've pushed this change and tested.